po/ar.po

```diff
@@ -3118,15 +3118,15 @@ msgid "Comets"
 msgstr "المذنبات"
 
 #. i18n: file: ../src/celestia/qt/preferences.ui:91
-#. i18n: ectx: property (text), widget (QCheckBox, spacecraftsCheck)
+#. i18n: ectx: property (text), widget (QCheckBox, spacecraftCheck)
 #. i18n: file: ../src/celestia/qt/preferences.ui:328
 #. i18n: ectx: property (text), widget (QCheckBox, spacecraftOrbitsCheck)
 #. i18n: file: ../src/celestia/qt/preferences.ui:548
 #. i18n: ectx: property (text), widget (QCheckBox, spacecraftLabelsCheck)
 #: ../src/celestia/qt/qtcelestiaactions.cpp:89
 #: ../src/celestia/qt/qtcelestiaactions.cpp:116 ../src/celestia/qt/rc.cpp:93
 #: ../src/celestia/qt/rc.cpp:174 ../src/celestia/qt/rc.cpp:240
-msgid "Spacecrafts"
+msgid "Spacecraft"
```

Comment:
see line 3927, already defined there, same for other po

Reply:
Should the line translating "Spacecrafts" be removed then?

this presents a problem in languages where the singular and plural of spacecraft are _not_ the same

Reply:
duplicates are not allowed in PO, which is making CI fail. personally I don't think we should use literal for msgid here because of problems like this.

for example this can be solved by renaming msgid to celestiaObjectSpacecraftSingular and celestiaObjectSpacecraftPlural. then in code, use celestiaObjectSpacecraftSingular/celestiaObjectSpacecraftPlural for a particular form. you should be able to find these references in code as _("Spacecraft")

Reply:
Celestia Origin .po file has both "Spacecraft" and "Spacecrafts", so I think I can just revert the .po file changes?

Reply:
im not entirely sure what you meant, all the po files have both Spacecraft and Spacecrafts. if you revert the change in po files, since in code they are all referred to as translation of Spacecraft, it will only be in singular form in other languages.